.github/workflows/python-service.yml

```diff
@@ -130,7 +136,62 @@ jobs:
           poetry run isort --filter-files .
 
       - name: Run tests
-        run: cd '${{ inputs.service }}'/src && poetry run python -m pytest
+        run: |
+          cd '${{ inputs.service }}'/src
+          if [[ ${{ env.RUN_COVERAGE }} = 'true' ]]; then
+            args='--cov --cov-report xml'
+          else
+            args=''
+          fi
+          poetry run python -m pytest $args
+
+      - name: Code Coverage Summary Report
+        uses: irongut/CodeCoverageSummary@v1.3.0
+        if: ${{ env.RUN_COVERAGE == 'true' }}
+        with:
+          filename: ${{ inputs.service }}/src/coverage.xml
+          badge: true
+          fail_below_min: 75
+          format: markdown
+          output: both
+
+      - name: Add Coverage PR Comment
+        uses: marocchino/sticky-pull-request-comment@v2
+        if: |
+          github.event_name == 'pull_request' ||
```

Comment:
Esta condicion deberia ser AND, no OR, no?
Porque si no se quiere RUN_COVERAGE no deberia saltar.